src/AutoMapper/XpressionMapper/XpressionMapperVisitor.cs

```diff
@@ -225,15 +224,10 @@ protected void FindDestinationFullName(Type typeSource, Type typeDestination, st
                 return;
             }
 
-            var typeMap = ConfigurationProvider.ResolveTypeMap(sourceType: typeDestination, destinationType: typeSource);//The destination becomes the source because to map a source expression to a destination expression,
-            //we need the expressions used to create the source from the destination 
-
-            if (typeMap == null)
-            {
-                throw QueryMapperHelper.MissingMapException(sourceType: typeDestination, destinationType: typeSource);
-            }
+            var typeMap = ConfigurationProvider.CheckIfMapExists(sourceType: typeDestination, destinationType: typeSource);//The destination becomes the source because to map a source expression to a destination expression,
+            //we need the expressions used to create the source from the destination 
 
-            PathMap pathMap = typeMap.FindPathMapByDestinationPath(sourceFullName);
+            PathMap pathMap = typeMap.FindPathMapByDestinationPath(destinationFullPath: sourceFullName);

```

Comment:
I don't think specifying the parameter name is helpful here (and below). The function name already makes the intention clear.

Reply:
ok - we can take it out.

Having said that it does make it clear that using the sourceFullName is not an error.